concent_api/core/migrations/0006_add_and_update_deadline_and_size.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.9 on 2018-06-19 09:37
+from __future__ import unicode_literals
+
+from django.db import migrations
+from django.db import models
+from common.helpers import deserialize_message
+from common.helpers import parse_timestamp_to_utc_datetime
+
+
+def populate_subtask_size_field(apps, _schema_editor):
+    Subtask = apps.get_model('core', 'Subtask')
+    for subtask in Subtask.objects.filter(result_package_size__isnull=True):
```

Comment:
What if `Subtask` for some reason will not have `report_computed_task` ? The line below will fail then. If that is desired behaviour then it is ok. If not, excluding `report_computed_task=None` should be added to filter.

Reply:
`report_computed_task` is not nullable.